modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -29,20 +29,21 @@ endif::[]
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Connection Timeout a| Number |  |  +++50000+++ | 
-| Socket Timeout a| Number |  |  +++50000+++ | 
-| Access Key a| String |  |  | x
-| Secret Key a| String |  |  | x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  | 
+| Connection Timeout a| Number |  +++The amount of time to wait (in milliseconds) when initially establishing a connection before giving up and timing out. A value of 0 means infinity, and is not recommended.+++ |  +++50000+++ | 
+| Socket Timeout a| Number |  +++The amount of time to wait (in milliseconds) for data to be transferred over an established, open connection before the connection is timed out. A value of 0 means infinity,and is not recommended.+++ |  +++50000+++ | 
+| Access Key a| String |  +++The access key provided by Amazon.+++ |  | x
+| Secret Key a| String |  +++The secret key provided by Amazon.+++ |  | x
+| Try Default AWSCredentials Provider Chain a| Boolean |  +++Set this field to true to obtain credentials from AWS environment, See: https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html+++ |  +++false+++ | 
+| Max Connections a| Number |  +++Sets the maximum number of allowed open HTTP connections.+++ |  | 
+| Region Endpoint a| String |  +++Set the topic region endpoint.+++ |  +++US_EAST_1+++ | 
 | Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy+++ |  | 
-| Host a| String |  |  | 
-| Port a| Number |  |  | 
-| Username a| String |  |  | 
-| Password a| String |  |  | 
-| Domain a| String |  |  | 
-| Workstation a| String |  |  | 
+| Host a| String |  +++The optional proxy host+++ |  | 
```

Comment:
Please add periods after the field descriptios.

Reply:
Done